src/lib.rs

```diff
@@ -144,7 +145,8 @@ mod context;
 pub mod constants;
 pub mod ecdh;
 pub mod key;
-pub mod schnorrsig;
```

Comment:
See comment below. Is it worth leaving `schnorrsig` in place and simply not exposing it here?